debian/jitsi-meet-web-config.postinst

```diff
@@ -182,36 +186,57 @@ case "$1" in
             echo "config.flags.receiveMultipleVideoStreams = true;" >> $JITSI_MEET_CONFIG
         fi
 
-        if [[ "$FORCE_NGINX" = "true" && ( -z "$JVB_HOSTNAME_OLD" || "$RECONFIGURING" = "true" ) ]] ; then
+        NGINX_CONF_PATH="/etc/nginx/sites-available/$JVB_HOSTNAME_OLD.conf"
+        OPENRESTY_CONF_PATH="/usr/local/openresty/nginx/conf/$JVB_HOSTNAME_OLD.conf"
```

Comment:
Sorry to insist, but the script is already complicated and if there is a room to improve I would love to see it. Thanks for your patience. 
Can we just drop `NGINX_CONF_PATH ` and `OPENRESTY_CONF_PATH ` and use the values directly in the if below? 
There is just one usage of `NGINX_CONF_PATH ` which I think anyway should be `NGX_COMMON_CONF_PATH`. WDYT, Is this correct?